client/my-sites/sidebar-unified/style.scss

```diff
@@ -501,7 +501,6 @@ $font-size: rem( 14px );
 			.sidebar__expandable-content {
 				display: block;
 				position: absolute;
-				top: 0;
```

Comment:
Does removing this allow the value to be inherited or is the new value actually `auto`? If so would it be better to make sure it remains as `auto` by declaring it explicitly?

Reply:
Reverted to `top: 0` as part of 5bdb744